commercetools-sdk-base/src/main/java/io/sphere/sdk/models/LocalizedString.java

```diff
@@ -31,6 +31,10 @@
 
     private static final Comparator<Map.Entry<Locale, String>> BY_LOCALE_COMPARATOR = (left, right) -> left.getKey().toString().compareTo(right.getKey().toString());
 
+    public static String getJsonCreatorName(){
```

Comment:
My IDE tells me that we don't need this method. Can we remove it or is it used via reflection?